xdist/dsession.py

```diff
@@ -1,4 +1,5 @@
 import difflib
+import importlib
```

Comment:
`importlib` is available only on Python >= 3.1, so unfortunately we can't use it like this. I suggest using `__import__` which is available in all Python versions we have to support.